apps/prairielearn/src/lib/db-types.ts

```diff
@@ -674,6 +674,18 @@ export const Lti13UserSchema = z.object({
 });
 export type Lti13User = z.infer<typeof Lti13UserSchema>;
 
+export const Lti13CourseInstanceSchema = z.object({
```

Comment:
Nit, let's keep this alphabetized per the comment way up at the top of this block.

Reply:
Addressed in b2ca77405